packages/db/postgres/tables.sql

```diff
@@ -199,6 +199,22 @@ CREATE TABLE IF NOT EXISTS pin_sync_request
 
 CREATE INDEX IF NOT EXISTS pin_sync_request_pin_id_idx ON pin_sync_request (pin_id);
 
+-- Tracks pinning requests from Pinning Service API
+-- TODO(paolo) this table should be considered for track all content stored on web3.storage
+CREATE TABLE IF NOT EXISTS pa_pin_request
```

Comment:
The naming here isn't great. 
But the aim is to rename this table to be called `pin_request` and the existing one to be `pin_replication_request`

Reply:
Defo, that's why we started with the not ideal name here.

Really keen to hear how you'd approach this, and we can create a follow up ticket to implement it? 
How does that sound?

Reply:
I think while this would be nice, we agreed to name the `table psa_pin_request`, so this is not strictly required.

Reply:
Rename done in in #810